Flow.Launcher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -139,6 +139,7 @@ public List<string> QuerySearchPrecisionStrings
             }
         }
 
+        public List<string> OpenResultModifiersList => new List<string> { "Alt", "Ctrl", "Ctrl+Alt" };
```

Comment:
great addition @gdziedzic.  Nitpick here, could we put the strings in Constant? I see them referenced more than once.

Also as @JohnTheGr8 mentioned, could we add an option to hide the modifier key display on main window please